experimental/LieAlgebras/src/WeylGroup.jl

```diff
@@ -51,10 +51,28 @@ function isomorphism(::Type{FPGroup}, W::WeylGroup; set_properties::Bool=true)
   return MapFromFunc(W, G, iso, isoinv)
 end
 
+@doc raw"""
+    permutation_group(W::WeylGroup; set_properties::Bool=true) -> PermGroup
```

Comment:
```suggestion
    permutation_group(W::WeylGroup) -> PermGroup
```
This is not supposed to be set by users. We need it for the tests since GAP may skip some checks we want it to do, if we set properties (that may not be true).
E.g., if we construct a non-isomorphic PermGroup here, which is of a different order; but we set the property to the known order from the Weyl group (thus giving GAP wrong information). In this case, GAP may state that both groups are of the same order, but in fact they aren't.

Reply:
I see. Thanks for the explanation!